METADATA.md

```diff
@@ -8,7 +10,32 @@
 | `website` | A website where the consumer can get their SMART Health Card or learn where they can get their SMART Health Card |
 | `help_line` | A phone number a consumer can call for assistance |
 | `issuer_type` | The type of issuer |
+| `locations` | A list of locations (see below for details) that the issuer is associated with |
+
+## Location Representation
+
+In order to best represent the reality of a SHC issuer issuing SHCs in multiple
+locations, an issuer can be associated to multiple country-state locations.
+
+This location representation is heavily inspired by the [FHIR `Address` type][fhir-address-type].
+
+| Attribute | Meaning |
+|-----------|---------|
 | `state` | Which state, province, territory, or other administrative division within a country the issuer represents |
```

Comment:
```suggestion
| `state` | The state, province, territory, or other administrative division within a country associated with the issuer |
```

Reply:
A pharmacy chain,for example, may be an issuer, but they do not "represent" the government.